crates/ruma-events/CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ Breaking changes:
 - The `event_id` field of `PreviousRoom` is now optional and deprecated. It has been removed in new
   room versions so clients should not rely on it. They can obtain it by requesting the
   `m.room.tombstone` event in the state of the predecessor.
+- The state key type `CallMemberStateKey` for `m.call.member` state events changed to allow any postfix
```

Comment:
This is not a breaking change because it is behind an `unstable-*` feature.

You can put this entry under "improvements".